### PR TITLE
Fix code samples in the README.md

### PR DESCRIPTION
**Pull request by Biruntha**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,22 +49,25 @@ twitter:Client twitterClient = new(twitterConfig);
...
-    string status = "Twitter endpoint test";
+twitter:Client twitterClient = new(twitterConfig);
 
-    twitter:Status|error result = twitterClient->tweet(status);
-    if (result is twitter:Status) {    
+public function main(string... args) {
+    string status = "Twitter endpoint test";
+    var result = twitterClient->tweet(status);
+    if (result is twitter:Status) {
+        //If successful, returns the tweet message or ID of the status.
```
Comment (MaryamZi) [inline README.md]:
```suggestion
        // If successful, print the tweet ID and text.
```

File: README.md [hunk 2]
```diff
@@ -49,22 +49,25 @@ twitter:Client twitterClient = new(twitterConfig);
...
         io:println("Tweet ID: ", result.id);
         io:println("Tweet: ", result.text);
     } else {
-        io:println("Error occurred on tweet attempt: ", result);
+        //Unsuccessful attempts return a Twitter error.
```
Comment (MaryamZi) [inline README.md]:
```suggestion
        // If unsuccessful, print the error returned.
```

File: README.md [hunk 3]
```diff
@@ -49,22 +49,25 @@ twitter:Client twitterClient = new(twitterConfig);
...
+        io:println(result);
```
Comment (MaryamZi) [inline README.md]:
```suggestion
        io:println("Error: ", result);
```

File: twitter/Module.md [hunk 4]
```diff
@@ -61,40 +61,49 @@ twitter:Client twitterClient = new(twitterConfig);
 
 The `tweet` function updates the current status. `status` is the text message of the status update.
 
-   `var tweetResponse = twitterClient->tweet(status);`
+   `var tweetResponse = twitterClient->tweet("This is a sample text message");`
```
Comment (MaryamZi) [inline twitter/Module.md]: Shall we say

```suggestion
   `var tweetResponse = twitterClient->tweet("This is a sample tweet");`
```

File: twitter/Module.md [hunk 5]
```diff
@@ -61,40 +61,49 @@ twitter:Client twitterClient = new(twitterConfig);
...
+    var tweetResponse = twitterClient->tweet(status);
+    if (tweetResponse is twitter:Status) {
+        //If successful, returns the tweet message or ID of the status.
+        string tweetId = <string> tweetResponse.id;
+        string text = tweetResponse.text;
+        io:println("Tweet ID: " + tweetId);
+        io:println("Tweet: " + text);
+    } else {
+        //Unsuccessful attempts return a Twitter error.
+        io:println(tweetResponse);
```
Comment (MaryamZi) [inline twitter/Module.md]: shall we replace this code snippet with the same one in the README.md?

File: twitter/Module.md [hunk 6]
```diff
@@ -61,40 +61,49 @@ twitter:Client twitterClient = new(twitterConfig);
...
-    io:println("Retweeted: " + tweetResponse.retweeted);
-} else {
-    io:println(tweetResponse);
-}
+    int tweetId = 1093833789346861057;
+    var tweetResponse = twitterClient->retweet(tweetId);
+    if (tweetResponse is twitter:Status) {
+        io:println("Retweeted: " + tweetResponse.retweeted);
+    } else {
+        io:println(tweetResponse);
```
Comment (MaryamZi) [inline twitter/Module.md]:
```suggestion
        io:println("Error: ", tweetResponse);
```

File: twitter/Module.md [hunk 7]
```diff
@@ -61,40 +61,49 @@ twitter:Client twitterClient = new(twitterConfig);
...
+    }
 ```
 
-The `search` function searches for tweets using a query string. It returns a `error` when unsuccessful.
+The `search` function searches for tweets using a query string. It returns a `Status[]` object if successful or `error` when unsuccessful.
```
Comment (MaryamZi) [inline twitter/Module.md]:
```suggestion
The `search` function searches for tweets using a query string. It returns a `Status[]` object if successful or an `error` if unsuccessful.
```

File: twitter/Module.md [hunk 8]
```diff
@@ -61,40 +61,49 @@ twitter:Client twitterClient = new(twitterConfig);
...
+    string queryStr = "twitter";
+    twitter:SearchRequest request = {
+        tweetsCount:"100"
+    };
+    var tweetResponse = twitterClient->search(queryStr, request);
+    if (tweetResponse is twitter:Status[]) {
+        io:println("Search Result :");
+        io:println(tweetResponse);
+    } else {
+        io:println(tweetResponse);
```
Comment (MaryamZi) [inline twitter/Module.md]:
```suggestion
        io:println("Error: ", tweetResponse);
```

File: twitter/Module.md [hunk 9]
```diff
@@ -61,40 +61,47 @@ twitter:Client twitterClient = new(twitterConfig);
...
 ```ballerina
-var tweetResponse = twitterClient->search(queryStr, searchRequest);
-if (tweetResponse is error) {
-    io:println(tweetResponse);
-} else {
-    io:println(tweetResponse);
-}
+    string queryStr = "twitter";
+    twitter:SearchRequest request = {
+        tweetsCount:"100"
```
Comment (MaryamZi) [inline twitter/Module.md]:
```suggestion
        tweetsCount: "100"
```

File: twitter/Module.md [hunk 10]
```diff
@@ -61,40 +61,47 @@ twitter:Client twitterClient = new(twitterConfig);
...
+    };
+    var tweetResponse = twitterClient->search(queryStr, request);
+    if (tweetResponse is twitter:Status[]) {
+        io:println("Search Result :");
```
Comment (MaryamZi) [inline twitter/Module.md]: Shall we add the space after `:` (and not before) and merge this line and the next to say 
```suggestion
        io:println("Search Result: ", tweetResponse);
```

Shall we also change the variable name to `searchResponse` ? 

Sorry, missed these in the previous review!

File: twitter/Module.md [hunk 11]
```diff
@@ -61,40 +61,46 @@ twitter:Client twitterClient = new(twitterConfig);
...
-var tweetResponse = twitterClient->retweet(tweetId);
-if (tweetResponse is twitter:Status) {
-    io:println("Retweeted: " + tweetResponse.retweeted);
-} else {
-    io:println(tweetResponse);
-}
+    int tweetId = 1093833789346861057;
+    var tweetResponse = twitterClient->retweet(tweetId);
+    if (tweetResponse is twitter:Status) {
+        io:println("Retweeted: " + tweetResponse.retweeted);
```
Comment (Shan1024) [inline twitter/Module.md]:
```suggestion
        io:println("Retweeted: ", tweetResponse.retweeted);
```

**Pull request merged**